src/api/api.js

```diff
@@ -0,0 +1,15 @@
+const URL = 'https://mate-api.herokuapp.com';
+
+export const getTodos = async() => {
+  const response = await fetch(`${URL}/todos`);
+  const result = await response.json();
```

Comment:
```suggestion
  const result = response.json();
```
response is already awaited